### PR TITLE
product card using  flex created

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**
- [DEMO LINK](https://roman-zherebetskyi.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://roman-zherebetskyi.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: .linthtmlrc.json [hunk 1]
```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
+    "align",
+    "background",
+    "bgcolor",
+    "border",
+    "frameborder",
+    "style"
+  ],
```
Comment (anton-iskryk) [inline .linthtmlrc.json:9]: There's no need to add this file. Also, don't add package-lock.json and package.json.

**Review by anton-iskryk — APPROVED** [review]
Good job! There's one thing for the future - you don't have to add files with the ending '.json'.

File: .linthtmlrc.json (hunk 1 above)
Comment (roman-zherebetskyi) [reply .linthtmlrc.json:9]: okay, got it! thx for the advice and thx for the review!

File: src/card.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+.card__image {
+  object-fit: cover;
+  max-width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  padding: 0 16px;
+}
```
Comment (Anna-Tkachenko) [inline src/card.css:24]: You've already specified paddings for the card block, why did you need to specify them one more time?

File: src/card.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  padding: 12px 0;
+  width: 100%;
+  background-color: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  transition: background-color 0.3s, color 0.3s;
+}
+
+.card__buy:hover {
+  border: 1px solid #00acdc;
```
Comment (Anna-Tkachenko) [inline src/card.css:87]: check point 9 in checklist

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,66 @@
...
+
+    <div class="card" data-qa="card">
+      <div class="card__inner">
+        <div class="card__img">
+          <img src="images/imac.jpeg" alt="product">
+        </div>
+
+        <h2 class="title card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <div class="card__product">Product code: 195434</div>
```
Comment (Anna-Tkachenko) [inline src/index.html:34]: use semantic tags

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,66 @@
...
+
+        <div class="card__star review">
+          <ul class="stars review__stars review__stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </ul>
+          <div class="review__reviews">Reviews: 5</div>
```
Comment (Anna-Tkachenko) [inline src/index.html:44]: use semantic tags

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,66 @@
...
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page__content">
+
+    <div class="card" data-qa="card">
+      <div class="card__inner">
```
Comment (Anna-Tkachenko) [inline src/index.html:27]: check this div, it's redundant wrapper, you can remove it

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,66 @@
...
+          </ul>
+          <div class="review__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price price">
+          <span class="price__title">Price:</span>
+          <strong class="price__price">$2,199</strong>
+        </div>
+
+        <div class="card__action action">
```
Comment (Anna-Tkachenko) [inline src/index.html:52]: check this wrapper, you can add all needed styles to the link and don't create additional blocks

File: src/stars.css [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(n) {
+  background: url(./images/star-active.svg) no-repeat center;
```
Comment (Anna-Tkachenko) [inline src/stars.css:20]: you should change only background-image, other background-properties you leave the same, so you don't repeat them here (no-repeat, center)

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+.card {
+  width: 200px;
+  border: 1px solid var(--color-light);
+  border-radius: var(--main-radius);
+}
+
+.card__inner {
+  padding: 32px 16px 15px;
+}
```
Comment (Anna-Tkachenko) [inline src/style.css:32]: styles for card block should be in card.css file

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
```
Comment (Anna-Tkachenko) [inline src/style.css:86]: check price block, do you actually need it? can you use it somewhere outside the card? maybe will be better to make it just the element of the card?

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+  margin: 0;
+}
+
+.stars {
+  width: 100px;
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  padding: 0;
+}
```
Comment (Anna-Tkachenko) [inline src/style.css:127]: styles for stars should be in stars.css file

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.review__stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```
Comment (Anna-Tkachenko) [inline src/style.css:145]: you should add styles for all possible modifiers, not only for stars--4

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 13]
```diff
@@ -1,13 +1,62 @@
...
+    <div class="card" data-qa="card">
+      <div class="card__logo">
+        <img
+          src="./images/imac.jpeg"
+          alt="logo"
+          class="card__image"
+        >
+      </div>
+      
+      <div class="card__content">
```
Comment (Anna-Tkachenko) [inline src/index.html:25]: this wrapper is redundant here, try to delete it

**Review by Anna-Tkachenko — APPROVED** [review]
